### PR TITLE
✨  Add WorkoutOverview

### LINKED ISSUE #5: Home
image

As a user, I would like to receive suggestions about which workouts are available and most popular. I would also like to see more detailed information on the proposed workouts.

- [ ] functionality
- [x] positioning
- [x] color

### LINKED ISSUE #6: Workout Overview
image

As a user, I would like to receive a detailed summary of a specific workout. This information can be, for example:
- Correct execution of the exercise
- muscle group
- equipment

After reading the description, I would like to be able to start the workout.

- [ ] functionality
- [ ] positioning
- [ ] color

### PR DESCRIPTION
**Pull request by tmh9x**
fixes #6 
https://fitdoc-app-tmh9x-issue6-ghsebp.herokuapp.com/storybook/?path=/story/page-workoutoverview--default

### CODE REVIEW

**Review by lajanzen — APPROVED** [review]
🚀

**Review by Niklas-Schaefers — APPROVED** [review]
lgtm 🚀

File: src/app/components/Header/Header.tsx [hunk 1]
```diff
@@ -1,17 +1,34 @@
...
-      <img className="w-7 h-7 mt-1 " src={thumbnail} />
-      {name}
-    </div>
+    <header className="grid grid-cols-3 pb-2 text-3xl border-b-2 text-secondary border-primary">
+      <div className="self-center">
+        {isBackButtonVisible && <BackButton thumbnail="back.png" />}
+      </div>
+      <div className="flex justify-center place-items-center">
+        <img className="w-7 h-7" src={thumbnail} />
+        {name}
```
Comment (phlgr) [inline src/app/components/Header/Header.tsx]: Is the name always the same or does it change based on the url visited?

Comment (tmh9x) [reply src/app/components/Header/Header.tsx:26]: sorry for late reply.. the name is based on the URL

**Pull request merged**